metrics/network/remote_network/README.md

```diff
@@ -34,3 +38,20 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-iperf3.sh -b -i eno1 -u tester -a 10.xxx.xxx.xxx
 
 ```
+
+## Running the `remote-networking-memory-smem.sh` test
+
+`remote-networking-memory-smem.sh` should run in host A. The script needs the following inputs to run:
+- Arguments of the test to run. In this case the argument is `-r` for RSS memory. All arguments should be
```

Comment:
Change the text for consistency and clarity:  Use the `-r` option for RSS memory.  This input is optional.